app/views/layouts/_nav.html.erb

```diff
@@ -6,6 +6,12 @@
     <% else %>
       <li> <%= link_to 'Login', new_user_session_path %> </li>
       <li> <%= link_to 'Register', new_user_registration_path %> </li>
+      <li><% unless current_user %>
+            <%= link_to "Sign in with Facebook", user_facebook_omniauth_authorize_path %>
+          <% else %>
+            <%= current_user[:name] %>
```

Comment:
I usually use `current_user.name` rather than the hash value markup. Not a blocker but a bit inconsistent.

Reply:
I agree and I'll fix it 👍